src/js/actions/tools.js

```diff
@@ -512,14 +513,6 @@ define(function (require, exports) {
                 }
                 
                 return this.transfer(selectTool, tool);
-            })
-            .catch(function (err) {
```

Comment:
There's no point in trying to recover if we fail to select an inferred tool above.